Description: Fix sha1sum for vcrun2013.
Origin: upstream, https://code.google.com/p/winetricks/source/detail?r=0d551c0cf409b1d1f6d6f3a096b0f159c6a4bba4
Bug: https://code.google.com/p/winetricks/issues/detail?id=464
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=775439
Last-Update: 2015-01-17
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: winetricks/src/winetricks
===================================================================
--- winetricks.orig/src/winetricks
+++ winetricks/src/winetricks
@@ -7318,7 +7318,9 @@ w_metadata vcrun2013 dlls \
 load_vcrun2013()
 {
     # http://www.microsoft.com/en-us/download/details.aspx?id=40784
-    w_download http://download.microsoft.com/download/2/E/6/2E61CFA4-993B-4DD4-91DA-3737CD5CD6E3/vcredist_x86.exe 18f81495bc5e6b293c69c28b0ac088a96debbab2
+    # 2014/07/26: 18f81495bc5e6b293c69c28b0ac088a96debbab2
+    # 2015/01/14: df7f0a73bfa077e483e51bfb97f5e2eceedfb6a3
+    w_download http://download.microsoft.com/download/2/E/6/2E61CFA4-993B-4DD4-91DA-3737CD5CD6E3/vcredist_x86.exe df7f0a73bfa077e483e51bfb97f5e2eceedfb6a3
 
     w_override_dlls native,builtin atl120 msvcp120 msvcr120 vcomp120
     cd "$W_CACHE"/vcrun2013
